src/screens/FeedScreen/RenderList.js

```diff
@@ -14,16 +14,17 @@ import {
   SOURCE_FEED_TAB
 } from '../../utils/constants';
 import {getCommentLength} from '../../utils/getstream';
-import {normalizeFontSizeByWidth} from '../../utils/fonts';
+import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
 import {COLORS} from '../../utils/theme';
 import dimen from '../../utils/dimen';
-
 import ShareUtils from '../../utils/share';
 import Content from './Content';
 import ContentLink from './ContentLink';
 import Header from './Header';
 import useFeed from './hooks/useFeed';
 import usePostHook from '../../hooks/core/post/usePostHook';
+import WriteComment from '../../components/Comments/WriteComment';
+import useWriteComment from '../../components/Comments/hooks/useWriteComment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `normalize` function is removed and `WriteComment` and `useWriteComment` are added. Ensure that `normalize` is not used anywhere else in the file, otherwise, it will cause a reference error. Also, make sure that `WriteComment` and `useWriteComment` are used appropriately in the code.
```diff
- import {normalizeFontSizeByWidth} from '../../utils/fonts';
+ import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
- import usePostHook from '../../hooks/core/post/usePostHook';
+ import usePostHook from '../../hooks/core/post/usePostHook';
+ import WriteComment from '../../components/Comments/WriteComment';
+ import useWriteComment from '../../components/Comments/hooks/useWriteComment';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->